client/js/components/tags/TagsList.vue

```diff
@@ -126,6 +127,10 @@ export default {
       TagTopic: 'items'
     }),
 
+    hasContent () {
+      return this.transformedCategories && this.transformedCategories.length > 0
```

Comment:
You can beautify this by:
```suggestion
      return this.transformedCategories?.length > 0
```

And: Are You sure, You have to test that at all? as far as i can see this.transformedCategories should always return an array, even if there are no TagTopics

Reply:
true, I changed it 138c965233086da83fe4d013c2e95bedccf8beb4